docusaurus.config.js

```diff
@@ -149,6 +149,11 @@ const config = {
       colorMode: {
         respectPrefersColorScheme: true,
       },
+      algolia: {
+        appId: 'DE7QZWOVO6',
+        apiKey: '6a95adc313e3bf3c03391b11dffe70fb',
```

Comment:
is it suitable to put `apiKey` here?

Reply:
<img width="523" alt="image" src="https://user-images.githubusercontent.com/14722250/180940874-e89ff669-a9bc-44b4-aaaa-7520bca847eb.png">

yes per the official docs